setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = fairly
+version = 0.1.0
+author = Serkan Girgin, Manuel Garcia Alvarez, Jose Urra Llanusa
+author_email = s.girgin@utwente.nl, m.g.garciaalvarez@tudelft.nl, m.g.garciaalvarez@tudelft.nl
+url = https://github.com/ITC-CRIB/fairly 
+description = A package to create, publish, and download research datasets from existing data repositories
+long_description =file: README.md
+long_description_content_type = text/markdown
+license =
+classifiers = MIT License 
+    Programming Language :: Python :: 3 
+    License :: OSI Approved :: MIT License 
+    Operating System :: OS Independent 
+
+
+[options]
+packages = find:
+package_dir=
+    =src/fairly
+
+[options.packages.find]
+where=src/fairly
+
+install_requires = 
+    python-dotenv==0.20.0    # We need this to configure our tokens for dev
+    requests_toolbelt==0.9.1 # This dependency was missing,
+
+[options.extras_require]
+dev =
+    ipython==8.4.0
+    pytest==7.1.2
+    black==22.6.0 # for code formmating
```

Comment:
I think fixing versions is not a good practice, as it enforces the package manager to uninstall existing packages if versions differ. We can skip versions, if they are not critical, or indicate minimum versions (>=xxx).